pep-0484.txt

```diff
@@ -1870,8 +1870,8 @@ The Typeshed Repo
 -----------------
 
 There is a shared repository where useful stubs are being collected
-[typeshed]_.  Note that stubs for a given package will not be included
-here without the explicit consent of the package owner.  Further
+[typeshed]_.  The package owner will be informed when stubs for a given
```

Comment:
```suggestion
[typeshed]_.  The package owner should be informed when stubs for a given
```

Reply:
Thanks for the review! Just to be clear, do you want me to make this change or just remove the sentence? Either option is fine with me.

Reply:
I was just coming back to suggest just dropping the sentence. :)